.github/workflows/docker_image.yml

```diff
@@ -0,0 +1,127 @@
+name: Docker build and Push
+
+# Trigger the workflow on pull request
+on:
+  pull_request:
+    types: [opened, synchronize, closed]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    
+    # Checkout the repository to the GitHub Actions runner
+    - uses: actions/checkout@v3
+
+
+    # Set up Python
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+
+    - name: Get branch name
+      id: branch_name
+      run: echo "::set-output name=branch::${GITHUB_HEAD_REF}"
+      
+    # Install yaml
+    - name: Install yaml and oyaml
+      run: |
+          pip install pyyaml
+          pip install oyaml
+
+    # Install linkml
+    - name: Install LinkML
+      run: pip install linkml
+
+    # Check if the PR description "Organization Name" has a value and matches the one in the model card
+    - name: Check PR description
+      if: github.event_name == 'pull_request' && github.event.pull_request.merged != true
+      id: check_pr_desc
+      run: |
+          python ./.github/workflows/check_pr_description.py
+      continue-on-error: true
+    
+    # Step 3: If the previous step failed, leave a comment and close the PR
+    - name: Leave a comment and close PR
+      if: steps.check_pr_desc.outcome != 'success' && github.event_name == 'pull_request' && github.event.pull_request.merged != true
```

Comment:
Same comment as above. Checking for event_name is redundant as well as replace `merged != true` with `opened == true`